examples/spacy/train.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+
+import random
+
+import spacy
+from spacy.util import minibatch, compounding
+
+import mlflow.spacy
+
+# training data
+TRAIN_DATA = [
+    ("Who is Shaka Khan?", {"entities": [(7, 17, "PERSON")]}),
+    ("I like London and Berlin.", {"entities": [(7, 13, "LOC"), (18, 24, "LOC")]}),
+]
+
+if __name__ == "__main__":
+    # Adaptation of spaCy example: https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py
+
+    # create blank model and add ner to the pipeline
+    nlp = spacy.blank("en")
+    ner = nlp.create_pipe("ner")
+    nlp.add_pipe(ner, last=True)
+
+    # add labels
+    for _, annotations in TRAIN_DATA:
+        for ent in annotations.get("entities"):
+            ner.add_label(ent[2])
+
+    params = {
+        'n_iter':100,
+        'drop': 0.5
+    }
+    mlflow.log_params(params)
+
+    nlp.begin_training()
+    for itn in range(params['n_iter']):
+        random.shuffle(TRAIN_DATA)
+        losses = {}
+        # batch up the examples using spaCy's minibatch
+        batches = minibatch(TRAIN_DATA, size=compounding(4.0, 32.0, 1.001))
+        for batch in batches:
+            texts, annotations = zip(*batch)
+            nlp.update(
+                texts,  # batch of texts
+                annotations,  # batch of annotations
+                drop=params['drop'],  # dropout - make it harder to memorise data
+                losses=losses,
+            )
+        print("Losses", losses)
+        mlflow.log_metrics(losses)
```

Comment:
```
mlflow.log_metrics(losses, step=itn)
```
Add `step` here?

Reply:
Sorry for the late reply after this PR is merged.

Reply:
@harupy no worries! That change makes sense, happy to review a follow-up PR that makes the change

Reply:
Hi, sure thing I'll make a follow-up PR with this small change ASAP. Thanks!